Description: simplify doxygen generation
 Remove the dot tool and all graph generation.
Author: Gauthier Quesnel <gauthier.quesnel@toulouse.inra.fr>
Origin: upstream
Forwarded: no
Last-Update: 2014-11-03
--- a/doxygen/doxygen.conf.in
+++ b/doxygen/doxygen.conf.in
@@ -1546,7 +1546,7 @@
 # the time of a run. So in most cases it will be better to enable call graphs
 # for selected functions only using the \callgraph command.
 
-CALL_GRAPH             = YES
+CALL_GRAPH             = NO
 
 # If the CALLER_GRAPH and HAVE_DOT tags are set to YES then
 # doxygen will generate a caller dependency graph for every global function
@@ -1554,7 +1554,7 @@
 # the time of a run. So in most cases it will be better to enable caller
 # graphs for selected functions only using the \callergraph command.
 
-CALLER_GRAPH           = YES
+CALLER_GRAPH           = NO
 
 # If the GRAPHICAL_HIERARCHY and HAVE_DOT tags are set to YES then doxygen
 # will graphical hierarchy of all classes instead of a textual one.
